tests/zkevm/test_worst_compute.py

```diff
@@ -314,7 +312,20 @@ def test_worst_modexp(
                 "090689D0585FF075EC9E99AD690C3395BC4B313370B38EF355ACDADCD122975B",
                 "12C85EA5DB8C6DEB4AAB71808DCB408FE3D1E7690C43D37B4CE6CC0166FA7DAA",
             ],
-            id="bn128_pairing",
+            id="bn128_two_pairings",
```

Comment:
Leaving this test as `two_pairings` to cover the point multiplication of the pairing check.